src/main/kotlin/team/mozu/dsm/adapter/out/item/persistence/mapper/ItemMapper.kt

```diff
@@ -0,0 +1,47 @@
+package team.mozu.dsm.adapter.out.item.persistence.mapper
+
+import org.springframework.stereotype.Component
+import team.mozu.dsm.adapter.out.`class`.entity.ClassJpaEntity
+import team.mozu.dsm.adapter.out.item.entity.ItemJpaEntity
+import team.mozu.dsm.adapter.out.organ.entity.OrganJpaEntity
+import team.mozu.dsm.domain.`class`.model.Class
+import team.mozu.dsm.domain.item.model.Item
+
+@Component
+class ItemMapper {
+    fun toModel(entity: ItemJpaEntity): Item {
+        return Item(
+            id = entity.id,
+            organId = entity.organ.id!!,
+            itemName = entity.itemName,
+            itemInfo = entity.itemInfo,
+            itemLogo = entity.itemLogo,
+            isDeleted = entity.isDeleted,
+            profit = entity.profit,
+            profitOg = entity.profitOg,
+            profitBenefit = entity.profitBenefit,
+            netProfit = entity.netProfit,
+            capital = entity.capital,
+            money = entity.money,
+            debt = entity.debt,
+            createdAt = entity.createdAt
+        )
+    }
+
+    fun toEntity(model: Item, organ: OrganJpaEntity): ItemJpaEntity {
+        return ItemJpaEntity(
+            organ = organ,
+            itemName = model.itemName,
+            itemLogo = model.itemLogo,
+            itemInfo = model.itemInfo,
+            isDeleted = model.isDeleted,
+            capital = model.capital,
+            debt = model.debt,
+            profit = model.profit,
+            profitOg = model.profitOg,
+            profitBenefit = model.profitBenefit,
+            netProfit = model.netProfit,
+            money = model.money
+        )
+    }
+}
```

Comment:
@Mapper를 사용해서 다음과 같이 수정하는 건 어떨까요?
```suggestion
@Mapper(component = "spring")
interface ItemMapper {
    fun toModel(entity: ItemJpaEntity): Item 

    fun toEntity(model: Item, organ: OrganJpaEntity): ItemJpaEntity
}
```

Reply:
4e2dd841532bbca6780eaff77e00ae112ebff08e